euth/documents/validators.py

```diff
@@ -0,0 +1,16 @@
+from django.core.exceptions import ValidationError
+
+from django.utils.translation import ugettext as _
+
+
+def single_document_per_module(module, pk=None):
+    from .models import Document
```

Comment:
I tried and it results in an import loop. Do you have any Idea how I could prevent that?